R/class-workbook-wrappers.R

```diff
@@ -707,6 +707,7 @@ wb_remove_row_heights <- function(wb, sheet = current_sheet(), rows) {
 #' wb$add_worksheet("Sheet 1", gridLines = FALSE)
 #'
 #' ## create plot objects
+#' if (find.packages("ggplot2", quiet = TRUE)) {
```

Comment:
Change to `if (requireNamespace("ggplot2", quietly = TRUE))`?  This is what CRAN might expect.

Reply:
Thanks, I'll try it tomorrow